backend/api/Controllers/RobotController.cs

```diff
@@ -234,16 +231,10 @@ public async Task<ActionResult<RobotResponse>> DeleteRobot([FromRoute] string id
                 return NotFound(errorMessage);
             }
 
-            robot.Status = robotStatus;
-
             try
             {
-                var updatedRobot = await robotService.Update(robot);
-                var robotResponse = new RobotResponse(updatedRobot);
-
-                logger.LogInformation("Successful PUT of robot to database");
-
-                return Ok(robotResponse);
+                await robotService.UpdateRobotStatus(id, robotStatus);
+                return Ok();
```

Comment:
RobotService.UpdateRobotStatus still seems to be returning a Robot object. What is the reason for not returning it here?

Reply:
I changed it back. The reason it was removed was because it should not return a robot object but I changed my mind midway through the PR.